opensrp-chw/src/kk/java/org/smartregister/chw/interactor/AncHomeVisitInteractorFlv.java

```diff
@@ -1024,17 +1027,22 @@ public String postProcess(String s) {
 
         @Override
         public String evaluateSubTitle() {
-            return MessageFormat.format(context.getString(R.string.chw_observations_evaluate_sub_title), anyone_presence);
+            if(!value.isEmpty()){
+                return MessageFormat.format(context.getString(R.string.chw_observations_evaluate_sub_title), value);
+            }else{
+                return value;
+            }
         }
 
         @Override
         public BaseAncHomeVisitAction.Status evaluateStatusOnPayload() {
-            if (!StringUtils.isBlank(anyone_presence))
-                if(anyone_presence.equalsIgnoreCase("yes") || anyone_presence.equalsIgnoreCase("ndio")){
+            if(!anyone_presence.isEmpty()){
+                if(anyone_presence.contains("yes")){
```

Comment:
Here the keys to check against for `anyone_presence` are
`anyone_present_yes`
`anyone_present_no`